proxy/src/telemetry/metrics/record.rs

```diff
@@ -84,8 +85,14 @@ impl Record {
                 })
             },
 
-            Event::TlsConfigReloaded =>
+            Event::TlsConfigReloaded(ref when) =>
                 self.update(|metrics| {
+                    let timestamp_secs = when
+                        .duration_since(UNIX_EPOCH)
+                        .expect("SystemTime before UNIX_EPOCH!")
+                        .as_secs()
+                        .into();
+                    metrics.tls_config_last_reload_seconds = Some(timestamp_secs);
```

Comment:
I filed https://github.com/runconduit/conduit/issues/1241 about factoring out the boilerplate that is duplicated between this and the other timestamp collection code.